coffeelint.json

```diff
@@ -27,5 +27,17 @@
   "braces_spacing": {
     "spaces": 0,
     "level": "error"
+  },
+  "space_operators": {
```

Comment:
Ah sorry, that's a good point (if only you could make this rule ignore defaults in parameters...) – I'll take this one out. How do the others look?